tests/testthat/test-missing_argument_linter.R

```diff
@@ -77,4 +77,28 @@ test_that("returns the correct linting", {
   expect_lint("alist(a =)",
     list(message = rex("Missing argument in function call.")),
     missing_argument_linter(c()))
+
+  expect_lint("fun(
```

Comment:
maybe add a comment saying why each test is here?

Reply:
done